src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -105,7 +105,7 @@ private async Task<string> GetBuildConfig(Uri url)
             var xrefMapApiEndpoint = GetXrefMapApiEndpoint(xrefEndpoint);
             if (docset.base_path != "/")
             {
-                xrefQueryTags.Add(docset.base_path);
+                xrefQueryTags.Add($"/{docset.base_path.TrimStart('/')}");
```

Comment:
we could not have a spec test for this since this is based on OP api docset info, it should be fine then